server/jetstream_cluster.go

```diff
@@ -785,6 +808,20 @@ func (js *jetStream) monitorCluster() {
 			if n.Leader() {
 				js.checkClusterSize()
 			}
+		case <-lt.C:
+			s.Debugf("Checking JetStream cluster state")
+			// If we have a current leader or had one in the past we can cancel this here since the metaleader
+			// will be in charge of all peer state changes.
+			// For cold boot only.
+			if n.GroupLeader() != _EMPTY_ || n.HadPreviousLeader() {
+				lt.Stop()
+				return
```

Comment:
Really? This will exit monitorCluster() that is used during mirroring, so will break it. Case in point test TestNoRaceJetStreamClusterSuperClusterMirrors failing locally..

Reply:
Yep should have been continue, on it.